public/app/features/alerting/unified/state/actions.ts

```diff
@@ -608,31 +608,29 @@ export const deleteMuteTimingAction = (alertManagerSourceName: string, muteTimin
           mute_time_intervals: muteIntervalsFiltered,
         };
 
-    if (config) {
-      const { mute_time_intervals: _, ...configWithoutMuteTimings } = config?.alertmanager_config ?? {};
-      withAppEvents(
-        dispatch(
-          updateAlertManagerConfigAction({
-            alertManagerSourceName,
-            oldConfig: config,
-            newConfig: {
-              ...config,
-              alertmanager_config: {
-                ...configWithoutMuteTimings,
-                route: config.alertmanager_config.route
-                  ? removeMuteTimingFromRoute(muteTimingName, config.alertmanager_config?.route)
-                  : undefined,
-                ...time_intervals_without_mute_to_save,
-              },
+    const { mute_time_intervals: _, ...configWithoutMuteTimings } = config?.alertmanager_config ?? {};
+    return withAppEvents(
```

Comment:
Change here was just to return the dispatched action so we can consistently chain off it inside useMuteTimings